Luminary116/SUBROUTINE_CALLS.agc

```diff
@@ -13,18 +13,18 @@
 ## Contact:     Ron Burkey <info@sandroid.org>.
 ## Website:     www.ibiblio.org/apollo/index.html
 ## Mod history: 2017-01-22 MAS  Created from Luminary 99.
-
+##              2017-03-01 f0n  Modified per scan 0089.jpg 
 ## NOTE: Page numbers below have not yet been updated to reflect Luminary 116.
```

Comment:
Can you delete this note now that the page numbers have been updated? Other than that, looks good!